buildscripts/zfs-driver/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:19.10
+FROM ubuntu:18.04
```

Comment:
Why was it set to 19.10? was there some dependency that we needed earlier?

Reply:
earlier there was libc issue, so it was set to the 19.10. but now with the root mount we don't need that.